README.md

```diff
@@ -1,18 +1,39 @@
-# @seasketch/geoprocessing
+# Geoprocessing
 
-A framework to create, combine, and publish geoprocessing `functions`, `datasources` and `client` reports for use in spatial planning tools.
+A framework for publishing low-cost and low-maintenance `geoprocessing` services and reports in the cloud.  Part of the [SeaSketch](https://seasketch.org/) ecosystem.
 
-Geoprocessing projects are designed to integrate with the [Seasketch](https://seasketch.org/) marine spatial planning platform, supporting continuous evaluation of user-designed planning areas (sketches) against science-based criteria.
+It's sweet spot is giving you fast spatial analysis functions that can be run at scale, over and over with different user input, with immediate report feedback, such as in support of an iterative spatial planning process.
```

Comment:
Trying to get even more specific here...
"The system works best for projects where the same analysis will be run many times with different input, and will be used by a large user group of diverse backgrounds such as in the case of marine spatial planning. It may be used to "operationalize" analysis that could have even been prototyped in other environments like R or Jupyter Notebook".

Reply:
"operationalize", I like that